.bandit.yml

```diff
@@ -6,8 +6,9 @@
 # If `tests` is empty, all tests are considered included.
 
 tests:
-# - B101
-# - B102
 
 skips:
-# - B101 # skip "assert used" check since assertions are required in pytests
+  - B101  # skip "assert used" check since assertions are required in pytests
+
+exclude:
+  - '**/test_*.py'
```

Comment:
Done.

Reply:
I unresolved this conversation because https://github.com/cisagov/XFD/pull/82/commits/e20fbfddf0c6b27e1d64bc75d2332a1fef3dde77 doesn't address the fact that you are still excluding test files in the bandit config file:
https://github.com/cisagov/XFD/blob/e20fbfddf0c6b27e1d64bc75d2332a1fef3dde77/.bandit.yml#L13-L14

Reply:
Understood. Fixed here: https://github.com/cisagov/XFD/pull/82/commits/bd2196eeb52c64827c835667fb9280e660e87f2b.

Reply:
Thanks!